### DISCUSSION TITLE
Some ideas for Windows version

### DISCUSSION THREAD

**Discussion by Mistarille224** [Abdenasser/neohtop#51] [Ideas]
GitHub recommended this project on my homepage. I downloaded it, opened it, and was amazed by it. The exquisite page is paired with a startup speed that is not inferior to the official task manager (even faster after testing).
So I'd be happy to ditch the official one entirely if the following was implemented:

- GPU status, Mentioned in #5
- A button to restart the process
- New right-click menu or ban it, the current content is completely meaningless
- Localized, for those speaking from non-English speakers

That's it, Windows users are always less picky, and it would be nice if the User column could show my username instead of a string of S-1-5...

**Comment by brainiacakinz** [comment]
ok
